pkg/resource/aws/aws_db_instance_ext.go

```diff
@@ -14,9 +14,15 @@ func (r *AwsDbInstance) NormalizeForState() (resource.Resource, error) {
 	if r.ApplyImmediately != nil && !*r.ApplyImmediately {
 		r.ApplyImmediately = nil
 	}
+	if r.CharacterSetName != nil && *r.CharacterSetName == "" {
```

Comment:
This fix need to be applied to the new infrastructure design in main branch @lotoussa @wbeuil

Reply:
Good catch will do it now !

Reply:
Done d427632